src/Microsoft.Diagnostics.Tracing.Eventing/Provider.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Diagnostics.Tracing.Eventing
+{
+    public struct Provider
+    {
+        public Provider(
+            string name,
+            ulong keywords = ulong.MaxValue,
+            EventLevel eventLevel = EventLevel.Verbose,
```

Comment:
This is more like a question but is there a reason we're making the provider as verbose as possible on default case?

Reply:
This has been the default for a while now, but I am OK with changing this.